src/main/java/com/zufar/onlinestore/payment/controller/PaymentController.java

```diff
@@ -52,4 +64,23 @@ public ResponseEntity<PaymentDetailsDto> getPaymentDetails(@PathVariable Long pa
         return ResponseEntity.ok()
                 .body(retrievedPayment);
     }
+
+    /**
+     * This endpoint is used only until we have an implementation of this logic on the frontend side.
+     * It will come in handy for testing the API.
+     */
+    @PostMapping("/method")
+    public ResponseEntity<String> getPaymentMethodId(@RequestBody CreatePaymentMethodDto paymentRequest) throws StripeException {
+        return ResponseEntity.ok().body(paymentMethodGetter.getPaymentMethodId(paymentRequest));
+    }
+
+    @PostMapping("/event")
+    public ResponseEntity<Void> paymentEventsProcess(@RequestBody String paymentIntentPayload, @RequestHeader("Stripe-Signature") String stripeSignatureHeader) throws SignatureVerificationException {
```

Comment:
move each method parameter to the new line, please
```
@RequestBody String paymentIntentPayload, 
@RequestHeader("Stripe-Signature") String stripeSignatureHeader
```

Reply:
> move each method parameter to the new line, please
> 
> ```
> @RequestBody String paymentIntentPayload, 
> @RequestHeader("Stripe-Signature") String stripeSignatureHeader
> ```

Done